client/rest-high-level/src/main/java/org/elasticsearch/client/indexlifecycle/OperationMode.java

```diff
@@ -56,4 +59,10 @@ public boolean isValidChange(OperationMode nextMode) {
     };
 
     public abstract boolean isValidChange(OperationMode nextMode);
+
+    static OperationMode fromString(String string) {
+        return EnumSet.allOf(OperationMode.class).stream()
+            .filter(e -> string.equalsIgnoreCase(e.name())).findFirst()
+            .orElseThrow(() -> new IllegalArgumentException(String.format(Locale.ENGLISH, "%s is not a valid operation_mode", string)));
```

Comment:
We use `Locale.ROOT` for as many things as we can, rather than the English locale here

Reply:
done